ShelterModuleTests/IntegrationTestSetupTests.cs

```diff
@@ -17,7 +18,10 @@ public sealed class IntegrationTestSetupTests
     public async Task ShouldUseUniqueDbPerTest(string shelterName)
     {
         using var connection = IntegrationTestSetup.CreateTestDatabase();
-        await using var context = IntegrationTestSetup.CreateDbContext(connection);
+
+        var testSetup = new IntegrationTestSetup();
+        using var scope = testSetup.Services.CreateScope();
+        await using var context = scope.ServiceProvider.GetRequiredService<PetShareDbContext>();
 
```

Comment:
That's not really what is tested in this test. We want to check if `IntegrationTestSetup.CreateTestDatabase()` creates connection strings to unique databases (because we need this in unit tests). Let's revert changes here.